api/users/views.py

```diff
@@ -505,6 +506,31 @@ def get_queryset(self):
         return Node.find(self.get_query_from_request())
 
 
+class UserPreprints(UserNodes):
+    required_read_scopes = [CoreScopes.USERS_READ]
```

Comment:
Oh, yeah. This says "if you can see profile data, you can also see data about preprints".

If we do reuse scopes, generally we should use the scope of a route that has the same serializer (same kind of data).